torch/csrc/jit/script/module.cpp

```diff
@@ -14,7 +14,7 @@ namespace torch {
 namespace jit {
 namespace script {
 
-static ModulePtr create_module_object(
+static ObjectPtr create_object_value(
```

Comment:
This looks like a mistaken find/replace renaming. Should still be called `create_module_object` given how it is used.